src/pages/patientView/patientHeader/GenomeDrivenDiagnosis.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from "react";
+import DefaultTooltip from "shared/components/defaultTooltip/DefaultTooltip";
+import {placeArrowBottomLeft} from "shared/components/defaultTooltip/DefaultTooltip";
+import SimpleTable from "shared/components/simpleTable/SimpleTable";
+import FontAwesome from "react-fontawesome";
+import {computed} from "mobx";
+
+export type GDDOutput = {
+    Pred1: string;
+    Pred2: string;
+    Pred3: string;
+    Conf1: string;
+    Conf2: string;
+    Conf3: string;
+    Diagnosed_Cancer_Type: string;
+    Diagnosed_Cancer_Type_Detailed: string;
+}
+
+// Genome Driven Diagnosis only works on these 22 cancer types
+export const GDD_CANCER_TYPES: string[] = [
+    "bladder cancer",
+    "breast cancer",
+    "cholangiocarcinoma",
+    "colorectal cancer",
+    "endometrial cancer",
+    "esophagogastric cancer",
+    "gastrointestinal stromal tumor",
+    "germ cell tumor",
+    "glioma",
+    "head and neck cancer",
+    "melanoma",
+    "mesothelioma",
+    "neuroblastoma",
+    "non-small cell lung cancer",
+    "ovarian cancer",
+    "pancreatic cancer",
+    "pancreatic neuroendocrine tumor",
+    "prostate cancer",
+    "renal cell cancer",
+    "small cell lung cancer",
+    "thyroid cancer",
+    "uveal melanoma",
+];
+
+// in percentage
+export const HIGH_CONFIDENCE_PREDICTION = 75;
+export const MEDIUM_CONFIDENCE_PREDICTION = 50;
+
+interface IGenomeDrivenDiagnosisProps {
+    prediction: GDDOutput;
+}
+
+export default class GenomeDrivenDiagnosis extends React.Component<IGenomeDrivenDiagnosisProps, {}> {
+    public render() {
+        return (
+            <DefaultTooltip
+                placement='bottomLeft'
+                trigger={['hover', 'focus']}
+                overlay={this.tooltipContent()}
+                destroyTooltipOnHide={false}
+                onPopupAlign={placeArrowBottomLeft}
+            >
+                <span style={{paddingLeft:2}}>
+                    {this.predictionContent()}
+                </span>
+            </DefaultTooltip>
+        );
+    }
+
+    @computed private get isInAllowedCancerTypes() {
```

Comment:
i would be careful with use of computed here.  The thing is, if the items reference inside are not observable, then computed will not reflect changes to them.   Unless you're using obviously observable properties, or there is a really strong case for optimization, I would avoid using computed because it's easy to mess up.